packages/cloud_firestore/cloud_firestore/example/integration_test/vector_value_e2e.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void runVectorValueTests() {
+  group('$VectorValue', () {
+    late FirebaseFirestore /*?*/ firestore;
```

Comment:
Might be worth adding e2e tests for potential edge cases, particularly for JS. For instance, not sure it would handle empty vector, single dimension, 2048 (maximum) dimensions, maximum dimensions + 1, very large values in vector, floats, negative values.